src/components/app/list-new/AppList.tsx

```diff
@@ -560,228 +401,155 @@ export default function AppList({ isSuperAdmin, appListCount, isArgoInstalled }:
-                                        isDisabled={dataStateType === AppListViewType.LOADING}
-                                        searchable
-                                        multi
-                                        type={AppListConstants.FilterType.APP_STATUS}
-                                        applyFilter={applyFilter}
-                                        onShowHideFilterContent={onShowHideFilterContent}
-                                        isFirstLetterCapitalize
-                                        dataTestId="app-status-filter"
+                                    <FilterSelectPicker
+                                        placeholder="App Status"
+                                        inputId="app-list-app-status-select"
+                                        options={appStatusFilters}
+                                        appliedFilterOptions={selectedAppStatus}
+                                        handleApplyFilter={handleUpdateFilters(AppListUrlFilters.appStatus)}
+                                        isDisabled={false}
+                                        isLoading={false}
                                     />
-                                    <span className="filter-divider" />
+                                    <div className="dc__border-right h-16" />
                                 </>
                             )}
-                            <Filter
-                                list={masterFilters.projects}
-                                labelKey="label"
-                                buttonText="Projects"
-                                placeholder="Search Project"
-                                isDisabled={dataStateType === AppListViewType.LOADING}
-                                searchable
-                                multi
-                                type={AppListConstants.FilterType.PROJECT}
-                                applyFilter={applyFilter}
-                                onShowHideFilterContent={onShowHideFilterContent}
-                                dataTestId="projects-filter"
+                            <FilterSelectPicker
+                                placeholder="Project"
+                                inputId="app-list-project-select"
+                                options={projectOptions}
+                                appliedFilterOptions={selectedProjects}
+                                handleApplyFilter={handleUpdateFilters(AppListUrlFilters.project)}
+                                isDisabled={appListFilterLoading || projectListLoading}
+                                isLoading={appListFilterLoading || projectListLoading}
+                                optionListError={appListFiltersError || projectListError}
+                                reloadOptionList={appListFilterResponse ? reloadAppFilters : reloadProjectOptions}
```

Comment:
This check seems ideally should be appListFiltersError ? reloadAppFilters : reloadProjectOptions